Sources/StreamVideoSwiftUI/Utils.swift

```diff
@@ -17,13 +18,9 @@ public class Utils {
 
 // MARK: - Utils + Default
 
-public extension Utils {
-    static var `default`: Utils = .init()
```

Comment:
Removing it as there is always supposed to be a single instance of Utils and this one is breaking this "contract".